### PR TITLE
add developer note deprecating ImportJob, Path in OMERO model

### PR DESCRIPTION
**Pull request by mtbc**
Documents https://github.com/openmicroscopy/openmicroscopy/pull/5517. Staged at https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/ws/src/omero/_build/html/developers/whatsnew.html.

### CODE REVIEW

File: omero/developers/whatsnew.rst [hunk 1]
```diff
@@ -19,20 +19,21 @@ Python BlitzGateway
 - See :doc:`/sysadmins/admins-with-restricted-privileges` to know which privileges are required for which actions.
 
 
-Graphs
-^^^^^^
-
-
 Java Gateway
```
Comment (hflynn) [inline omero/developers/whatsnew.rst:22]: This heading can be removed too if there's no content for it

Comment (mtbc) [reply omero/developers/whatsnew.rst:22]: I hope for content: https://trello.com/c/szy4IbD2/156-document-gateway-enhancements.

Comment (hflynn) [reply omero/developers/whatsnew.rst:22]: 👍

**Comment by hflynn** [comment]
Code PR merged, content renders well and links work as expected, merging thanks

**Pull request merged**